src/cata_tiles.cpp

```diff
@@ -1390,9 +1390,9 @@ void cata_tiles::draw( const point &dest, const tripoint &center, int width, int
             // Add scent value to the overlay_strings list for every visible tile when
             // displaying scent
             if( g->display_overlay_state( ACTION_DISPLAY_SCENT ) && !invis ) {
-                const int scent_value = get_scent().get( {temp.x, temp.y, center.z} );
+                const int scent_value = get_scent().get( pos );
                 if( scent_value > 0 ) {
-                    overlay_strings.emplace( player_to_screen( point( temp.x, temp.y ) ) + half_tile,
+                    overlay_strings.emplace( player_to_screen( point( temp ) ) + half_tile,
```

Comment:
```suggestion
                    overlay_strings.emplace( player_to_screen( temp ) + half_tile,
```
`temp` is already of type `point`, so you don't need to wrap it